src/components/History/History.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import Card from '../Card/Card';
+import { CleanedArticle } from '../../Models';
+import '../Feed/Feed.css'
+
+interface HistoryProps {
+  history: CleanedArticle[] | any
+  updateHistory: (localHistory: string[]) => void;
+}
+
+const History = ({ history, updateHistory }: HistoryProps): JSX.Element => {
+
+  const articleCards = history.map((article: CleanedArticle) => {
+    return (
+      <Card
+        title={ article.title }
+        image={ article.multimedia.url }
+        key={ article.title }
+        sentiment={ article.sentiment }
+        id= { article.id }
+        updateHistory = { updateHistory }
+      />
+    )
+  })
+
+  return (
+    <>
+      <h1>History</h1>
+      <div className="articles-container">
```

Comment:
Would the styling change if we do not include this `div`?

Reply:
This is the next piece of history I'm working on. I copied this structure from Feed.tsx before it was cleaned up.